package.json

```diff
@@ -23,9 +23,7 @@
     "build-viewer": "cd ./lighthouse-viewer && yarn build",
     "clean": "rimraf *.report.html *.report.dom.html *.report.json *.screenshots.html *.screenshots.json *.devtoolslog.json *.trace.json || true",
     "lint": "[ \"$CI\" = true ] && eslint --quiet -f codeframe . || eslint .",
-
     "smoke": "node lighthouse-cli/test/smokehouse/run-smoke.js",
```

Comment:
Don't know if these whitespace trims are needed

Reply:
yeah, these are (annoyingly) trimmed automatically when running one of several different yarn/npm commands, but since no functional changes to the file can just remove from the PR

Reply:
This was required because there was a conflict reported in `package.json` by Github.